lib/Conversion/ImportVerilog/Structure.cpp

```diff
@@ -25,11 +25,21 @@ struct MemberVisitor {
   MemberVisitor(Context &context, Location loc)
       : context(context), loc(loc), builder(context.builder) {}
 
-  /// Skip semicolons.
+  // Skip empty members (stray semicolons).
   LogicalResult visit(const slang::ast::EmptyMemberSymbol &) {
     return success();
   }
 
+  // Skip members that are implicitly imported from some other scope for the
+  // sake of name resolution, such as enum variant names.
+  LogicalResult visit(const slang::ast::TransparentMemberSymbol &) {
+    return success();
+  }
+
+  // Skip typedefs.
+  LogicalResult visit(const slang::ast::TypeAliasType &) { return success(); }
```

Comment:
Yes exactly. They stay in here for the sake of completeness, and possibly if you tried to use the AST before elaboration (which I'm not sure is actually an option), but many of them don't need to generate any ops because they have been folded into other nodes (either as type in the case of typedefs, or as resolved name in the case of import statements).